src/renderers/dom/server/__tests__/ReactServerRendering-test.js

```diff
@@ -400,4 +400,85 @@ describe('ReactServerRendering', function() {
       expect(markup.indexOf('hello, world') >= 0).toBe(true);
     });
   });
+
+  it('warns with a no-op when an async setState is triggered', function() {
+    var Component = React.createClass({
+      componentWillMount: function() {
+        this.setState({text: 'hello'});
+        setTimeout(() => {
+          this.setState({text: 'error'});
+        });
+      },
+      render: function() {
+        return <div onClick={() => {}}>{this.state.text}</div>;
+      },
+    });
+
+    spyOn(console, 'error');
+    ReactServerRendering.renderToString(<Component />);
+    jest.runOnlyPendingTimers();
+    expect(console.error.calls.count()).toBe(1);
+    expect(console.error.calls.mostRecent().args[0])
+    .toBe(
```

Comment:
Nit: let’s put `.toBe(` at the end of previous line. Same for all other occurrences.